To: vim_dev@googlegroups.com
Subject: Patch 7.3.204
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.204 (after 7.3.201)
Problem:    Compiler warning.
Solution:   Add type cast. (Mike Williams)
Files:	    src/misc1.c


*** ../mercurial/vim73/src/misc1.c	2011-05-25 15:16:06.000000000 +0200
--- src/misc1.c	2011-05-25 17:25:10.000000000 +0200
***************
*** 7657,7663 ****
  			     * not the one from "if () {". */
  			    if (*l == '}')
  				curwin->w_cursor.col =
! 						   (l - ml_get_curline()) + 1;
  
  			    if ((trypos = find_start_brace(ind_maxcomment))
  								       == NULL
--- 7657,7663 ----
  			     * not the one from "if () {". */
  			    if (*l == '}')
  				curwin->w_cursor.col =
! 					  (colnr_T)(l - ml_get_curline()) + 1;
  
  			    if ((trypos = find_start_brace(ind_maxcomment))
  								       == NULL
*** ../vim-7.3.203/src/version.c	2011-05-25 17:06:16.000000000 +0200
--- src/version.c	2011-05-25 17:29:32.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     204,
  /**/

-- 
In a world without walls and borders, who needs windows and gates?

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
